File: SpringApp1/src/main/java/com/beans/Employee.java
package com.beans;

public class Employee {

	private int empno;
	private String ename;
	private String job;
	private float salary;
	
	public int getEmpno() {
		return empno;
	}
	public void setEmpno(int empno) {
		this.empno = empno;
		//System.out.println("Emp Number");
	}
	public String getEname() {
		return ename;
	}
	public void setEname(String ename) {
		this.ename = ename;
		//System.out.println("Emp Name");
	}
	public String getJob() {
		return job;
	}
	public void setJob(String job) {
		this.job = job;
	}
	public float getSalary() {
		return salary;
	}
	public void setSalary(float salary) {
		this.salary = salary;
	}
	
	
}

File: WebDesign/AngApps/FormsApp/src/app/employee.ts
export class Employee {

    id:number=0;
    ename:string = "";
    job:string = "";
    salary:number = 0.0;

    constructor(id:number, ename:string, job:string, sal:number){
        this.id = id;
        this.ename = ename;
        this.job = job;
        this.salary = sal;
    }
}
